src/Microsoft.AspNet.Mvc.Core/ApplicationModels/DefaultActionModelBuilder.cs

```diff
@@ -266,6 +268,18 @@ private bool IsIDisposableMethod(MethodInfo methodInfo, TypeInfo typeInfo)
             AddRange(actionModel.ActionConstraints, attributes.OfType<IActionConstraintMetadata>());
             AddRange(actionModel.Filters, attributes.OfType<IFilter>());
 
+            var enableCors = attributes.OfType<IEnableCorsMetadata>().SingleOrDefault();
```

Comment:
I noticed the authorize code only has AuthorizeAttribute, but no associated interface. Those should probably be consistent, maybe file a bug to add `IAuthorizeAttribute` along with the `AuthorizeAttribute`

Also, if we do marker interfaces in favor of a well-known attribute clas it should be named something like `IEnableCorsAttribute`.